runtime/compiler/control/CompilationRuntime.hpp

```diff
@@ -1142,6 +1142,7 @@ class CompilationInfo
    uint64_t               _lastCompilationsShouldBeInterruptedTime; // RAS
 // statistics
    int32_t                _statsOptLevels[numHotnessLevels]; // will be zeroed with memset
+   int32_t                _statsRemoteOptLevels[numHotnessLevels];
```

Comment:
Since this code is only used for JITServer we need to protect it with 
`#if defined(J9VM_OPT_JITSERVER)`
See other parts of the code for an example

Reply:
Fields should start with underscores.
However, in this case I don't think we need `totalNumRemoteCompilations` or `totalMsgCount`. They can be computed on demand